### PR TITLE
Implemented binary serialization for create_unique

### PR DESCRIPTION
**Pull request by iNemesis21**
Hello,

In this PR I've solved feature #69. Also enabled the binary serialization/deserialization test from quickcheck, which passes.

### CODE REVIEW

File: src/transactions/src/create_unique.rs [hunk 1]
```diff
@@ -82,6 +83,316 @@ impl CreateUnique {
         }
     }
 
+    /// Serializes the transaction struct to a binary format.
+    ///
+    /// Fields:
+    /// 1) Transaction type(9)  - 8bits
+    /// 2) Fee length           - 8bits
+    /// 3) Meta Existence Map   - 5byte binary
```
Comment (octavonce) [inline src/transactions/src/create_unique.rs]: We should use a bitmask here so it will use one byte instead of five.

**Review by octavonce — CHANGES_REQUESTED** [review]

**Comment by iNemesis21** [comment]
Hi @octavonce 
Per your suggestion, I've changed the Meta Existence Map to a BitMask.

**Comment by octavonce** [comment]
@iNemesis21 This looks great! Can you also uncomment this line so we hook the transaction serialization to `Heartbeat` serialization? https://github.com/purpleprotocol/purple/blob/master/src/events/src/heartbeat.rs#L342

**Comment by iNemesis21** [comment]
@octavonce Done in last commit.

**Review by octavonce — APPROVED** [review]

**Pull request merged**